client/src/components/App.jsx

```diff
@@ -1,21 +1,42 @@
+/* eslint-disable import/extensions */
 import React from 'react';
-import Header from './Header.jsx';
+import axios from 'axios';
+import Summary from './Summary.jsx';
+import Reviews from './Reviews.jsx';
+import styles from './style/app.css';
 
 class App extends React.Component {
   constructor() {
     super();
     this.state = {
-      states: '',
+      reviews: '',
```

Comment:
Is reviews supposed to be a string or an array?

Reply:
reviews is an array

Reply:
I just used ' ' as a placeholder because it gets populated with the get request later

Reply:
Can you change it to use [] as a placeholder instead for clarity?

Reply:
yep!